history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -951,15 +1045,29 @@ not only in word, but in deed also,
 
 I write this at the request of certain of my friends called Quakers.
 
-W+++.+++ SEWEL
+[.signed-section-signature]
+W+++.+++ Sewel
 
+[.signed-section-context-close]
 Amsterdam, the 23rd day of the month called April, 1679.
 
-Some questions presented to the Catholics, and the Pope of Rome,
-as the supreme head of their church, and commended to their consideration by George Fox.
+--
+
+[.embedded-content-document.treatise]
```

Comment:
I wasn't sure if I should call this a treatise.

Reply:
maybe let's go with `address` instead of treatise.